src/modules/client/pages/ni/courses/Dashboard.vue

```diff
@@ -1,6 +1,14 @@
 <template>
   <q-page class="client-background" padding>
     <ni-title-header title="La formation Compani dans ma structure" />
+    <div v-if="linkRequests.length" class="bg-yellow-100 q-my-md q-pa-md">
+      <p class="text-orange-900 text-weight-bold">Demandes de rattachement à ma structure</p>
+      <div class="request-container">
+        <div v-for="request in linkRequests" :key="request._id">
+          <company-link-request-cell :user="request.user" @click="refreshCompanyLinkRequests" />
+        </div>
```

Comment:
je pense que tu n'as pas besoin de la `div` et que tu peux mettre directement le `v-for` sur `<company-link-request-cell`

Reply:
done